Testscripts/Linux/docker_python_app.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# Description:
+#   Script to run the hello-world python app in docker.
+#
+########################################################################
+
+DOCKER_FILENAME="Dockerfile"
+
+PYTHON_PROG_NAME="helloworld.py"
+
+CONTAINER_NAME="pythonapp"
+CONTAINER_IMAGE_NAME="pythonappimage"
+
+STRING_IDENTIFIER="Hello world from python"
+
+# Function to generate python program which will run
+# inside docker
+GeneratePythonProgram()
+{
+cat << EOF > $PYTHON_PROG_NAME
+print("$STRING_IDENTIFIER")
+EOF
+    [[ ! -f $PYTHON_PROG_NAME ]] && return 1
+    return 0
+}
+
+# Function to generate docker file
+GenerateDockerFile()
+{
+cat << EOF > $DOCKER_FILENAME
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install python3 -y
```

Comment:
There is package installation function in utils.sh.  Also you may think if python3 is installed but python link might be missing. You should check those conditions

Reply:
This installation is for the package inside containers and need to be a part of docker file.